language-design.md

```diff
@@ -69,7 +155,20 @@ The Yang and Rabkin article talks mainly about general-purpose languages. In
 what ways do the themes apply to the study and creation of DSLs?
 
 **Response**
-
+One thing we believe is that DSLs are just as susceptible as general-purpose
+programming languages to the biases and stereotypes people impose. DSLs, by
+design, are not as big and popular as other languages such as Java and Python.
+Because of this, they might be perceived as inferior due to biases from the
+larger languages' communites. In addition, communities for DSLs are generally 
+much smaller and can make DSLs seem weak socially.
+
+We are both curious how DSLs fit into the world. To what extent are they
+academic/research topics, and to what extent are they currently used tools in
+the world? And in considering the DSLs that are out there being used, to whom
+are they given to? We believe that DSLs are available both in high-end academia
+as well as precollege. For example, Scratch is a DSL (probably) that is used
+in primary to secondary school. Because of this polarization, we believe the
+topics in the article apply to DSLs.
```

Comment:
Definitely agree that the same stereotypes affect DSLs. Interesting that you compare the use of a programming language or a DSL as an 'either or' situation. What if they are used together? For instance, using jQuery with JavaScript. Is the size of the DSLs and the host language's community still an issue?

To explore the question of how DSLs fit in the world, it seems that DSLs are created to fill any perceived void within any domain. As long as the cost of creating and maintaining it is outweighed by the benefit in productivity, they will exist.